src/utils.js

```diff
@@ -55,18 +56,16 @@ const ESCAPE_LOOKUP = {
 
 const ESCAPE_REGEX = /[&><"']/g;
 
-function escaper(match) {
+// Type-safe only if the input string matches ESCAPE_REGEX.
+function escaper(match: string): string {
```

Comment:
It's possible that `escaper` could return `undefined`.

Reply:
Not given the comment I'd added above `escaper()`. It's used only for `escape()`. In fact, I removed this function and just inlined it where it's used.